enumerables.rb

```diff
@@ -0,0 +1,214 @@
+# p a.my_map(&test_proc)
+# p a.my_inject (10) { |a, b| a + b }
+# p multiply_els([2, 4, 5])
+
+# p %w{ant bear cat}.my_none? { |word| word.length == 5 } #=> true
+# p %w{ant bear cat}.my_none? { |word| word.length >= 4 } #=> false
+# p %w{ant bear cat}.my_none?(/d/)                        #=> true
+# p [1, 3.14, 42].my_none?(Float)                         #=> false
+# p [].my_none?                                           #=> true
+# p  [nil].my_none?                                        #=> true
+# p [nil, false].my_none?                                 #=> true
+# p [nil, false, true].my_none?                           #=> false
+
+# p %w[ant bear cat].my_all? { |word| word.length >= 3 } #=> true
+# p %w[ant bear cat].my_all? { |word| word.length >= 4 } #=> false
+# p %w[ant bear cat].my_all?(/t/)                        #=> false
+# p [1, 2i, 3.14].my_all?(Numeric)                       #=> true
+# p [nil, true, 99].my_all?                              #=> false
+# p [].my_all?                                           #=> true
+
+# p %w[ant bear cat].my_any? { |word| word.length >= 3 } #=> true
+# p %w[ant bear cat].my_any? { |word| word.length >= 4 } #=> true
+# p %w[ant bear cat].my_any?(/d/)                        #=> false
+# p [nil, true, 99].my_any?(Integer)                     #=> true
+# p [nil, true, 99].my_any?                              #=> true
+# p [].my_any?                                           #=> false
+
+# p [3, 5, 7, 11].my_all?(&:odd?) # => true
+# p [-8, -9, -6].my_all? { |n| n < 0 } # => true
+# p [3, 5, 8, 11].my_all?(&:odd?) # => false
+# p [-8, -9, -6, 0].my_all? { |n| n < 0 } # => false
+# # test cases required by tse reviewer
+# p [1, 2, 3, 4, 5].my_all? # => true
+# p [1, 2, 3].my_all?(Integer) # => true
+# p %w[dog door rod blade].my_all?(/d/) # => true
+# p [1, 1, 1].my_all?(1) # => true
+
+# p [7, 10, 4, 5].my_any?(&:even?) # => true
+# p %w[q r s i].my_any? { |char| 'aeiou'.include?(char) } # => true
+# p [7, 11, 3, 5].my_any?(&:even?) # => false
+# p %w[q r s t].my_any? { |char| 'aeiou'.include?(char) } # => false
+# # test cases required by tse reviewer
+# p [3, 5, 4, 11].my_any? # => true
+# p "yo? #{[nil, false, nil, false].my_any?}" # => false
+# p [1, nil, false].my_any?(1) # => true
+# p [1, nil, false].my_any?(Integer) # => true
+# p %w[dog door rod blade].my_any?(/z/) # => false
+# p [1, 2, 3].my_any?(1) # => true
+
+# p [3, 5, 7, 11].my_none?(&:even?) # => true
+# p [1, 2, 3, 4].my_none?{|num| num > 4} #=> true
+# p [nil, false, nil, false].my_none? # => true
+# p %w[sushi pizza burrito].my_none? { |word| word[0] == 'a' } # => true
+# p [3, 5, 4, 7, 11].my_none?(&:even?) # => false
+# p %w[asparagus sushi pizza apple burrito].my_none? { |word| word[0] == 'a' } # => false
+# # test cases required by tse reviewer
+# p [1, 2, 3].my_none? # => false
+# p [1, 2, 3].my_none?(String) # => true
+# p [1, 2, 3, 4, 5].my_none?(2) # => false
+# p [1, 2, 3].my_none?(4) # => true
```

Comment:
Kindly delete or move to another branch all of this debugging code. :hammer_and_wrench: :+1: